contracts/ConversionRates2.sol

```diff
@@ -21,8 +21,8 @@ contract ConversionRates2 is ConversionRates {
         public
         onlyOperator
     {
-        require(xBuy.length == yBuy.length);
-        require(xSell.length == ySell.length);
+        require(xBuy.length + 1 == yBuy.length);
+        require(xSell.length + 1 == ySell.length);
         require(xBuy.length <= MAX_STEPS_IN_FUNCTION);
```

Comment:
Max steps should have a new value.
create a new const .
16 steps.

actually now max steps should be compared to y length. and not x length.

Reply:
Change max step func to 16 and compare with y length